custom_components/tuya_local/devices/tuya_smart_zigbee_door_sensor.yaml

```diff
@@ -0,0 +1,27 @@
+name: Tuya Smart Zigbee Door Sensor
+products:
+  - id: 7jIGJAymiH8OsFFb
+    name: Tuya Smart Zigbee Door Sensor
+primary_entity:
+  entity: binary_sensor
+  class: door
+  dps:
+    - id: 101
+      type: boolean
+      name: sensor
+secondary_entities:
+  - entity: sensor
+    class: battery
+    category: diagnostic
+    dps:
+      - id: 103
+        type: integer
+        name: sensor
+        optional: true
+        unit: "%"
+        class: measurement
+        mapping:
+          - dps_val: low
```

Comment:
If this is an integer, it should not geet values "low" and "high".  I don't understand why this mapping is here.

Reply:
@make-all learn as I go, so bear with me please. Updated.
If there is a doc that I am missing - please let me know. I checked with other files, seems like this is how it must be done.